src/roi/Roi.ts

```diff
@@ -13,11 +13,16 @@ import { RoiMap } from './RoiMapManager';
 import { getBorderPoints } from './getBorderPoints';
 import { getMask, GetMaskOptions } from './getMask';
 import { Ellipse, getEllipse } from './properties/getEllipse';
-
+/**
+ * Interface for borders of ROI.
```

Comment:
ditto for "interface"

Since border is exposed (returned by some public property of ROI), then this interface should have a more specific name (maybe `ROIBorder`) and be exported

Reply:
"Interface" removed from the description. Change of the name of the interface should probably be done on a different branch.